components/dashboard/src/hooks/use-analytics-tracking.ts

```diff
@@ -27,6 +27,11 @@ export const useAnalyticsTracking = () => {
 
         return history.listen((location: any) => {
             const path = window.location.pathname;
+            // Do not track things like changes to the hash
```

Comment:
Why not? What broke?

Reply:
Nothing broke per se, but we spam the analytics endpoint with the same path when doing things like changing the currently selected task in prebuilds.

Reply:
The comment is a bit misleading: we never tracked the hash, but we always responded to changes to it. This makes it not respond to those events anymore

Will update that